### ISSUE TITLE
[Bug]: All Dividers consumed on map refinement

### ISSUE THREAD

**Issue by jhblanck** [MelvorIdle/melvoridle.github.io#3661]
### Describe the bug

All Dividers consumables are consumed when refining a map.

### Reproduction Steps

1. Equip at least 2 Dividers.
2. Go to the Dig Site Map Refinement Sub-Menu.
3. Select one of the available refinements.
4. Observe that all equipped Dividers have been consumed.

### Expected behaviour

After a single refinement, only 1 Divider should have been consumed.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [X] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Steam
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.2

### Game Subversion

?7832

### Are you using any mods?

Yes

**Comment by jhblanck** [comment]
For clarity, the only mod I am using is the [Myth] Combat Simulator.

**Comment by Flurryous** [comment]
I'm also using the same mod for my files. This one's kind of weird because I've noticed it happening sometimes too. According to my statistics page, I refined 69 times but I seem to have used 139 dividers somehow out of the 186 I had total. I noticed that all my dividers disappeared when I was refining a Bazaar map, so I reloaded that save to check out what's going on, but when I refined the same map again, only one divider was used each refinement. This is a weird bug that seems to only happen sometimes for me, so it might be difficult to replicate.
